components/script/dom/extendablemessageevent.rs

```diff
@@ -123,6 +123,21 @@ impl ExtendableMessageEvent {
         );
         Extendablemessageevent.upcast::<Event>().fire(target);
     }
+
+    pub fn dispatch_error(target: &EventTarget, scope: &GlobalScope, message: HandleValue) {
+        let ExtendableMsgEvent = ExtendableMessageEvent::new(
+            scope,
+            atom!("messageerror"),
+            false,
+            false,
+            message,
+            DOMString::new(),
+            DOMString::new(),
+            Vec::<DomRoot<MessagePort>>::new(),
+        );
+
```

Comment:
nit: I don't think we need the newline here(not something fmt will fix).

Reply:
I'll clean it up. Thank you!